pkg/controllers/dashboard/fleetcharts/controller.go

```diff
@@ -61,10 +62,25 @@ func (h *handler) onSetting(key string, setting *v3.Setting) (*v3.Setting, error
 			"systemDefaultRegistry": settings.SystemDefaultRegistry.Get(),
 		},
 	}
-	return setting, h.manager.Ensure(fleetChart.ReleaseNamespace, fleetChart.ChartName,
-		map[string]interface{}{
-			"apiServerURL": settings.ServerURL.Get(),
-			"apiServerCA":  settings.CACerts.Get(),
-			"global":       systemGlobalRegistry,
-		})
+
+	fleetChartValues := map[string]interface{}{
+		"apiServerURL": settings.ServerURL.Get(),
+		"apiServerCA":  settings.CACerts.Get(),
+		"global":       systemGlobalRegistry,
+	}
+
+	gitjobChartValues := map[string]interface{}{}
+
+	if envVal, ok := os.LookupEnv("HTTP_PROXY"); ok {
+		gitjobChartValues["proxy"] = envVal
+	}
+	if envVal, ok := os.LookupEnv("NO_PROXY"); ok {
```

Comment:
Did you miss `HTTPS_PROXY`?

Reply:
Nvm, I think they have the same setting.

Reply:
Good point, technically they can be different.

Reply:
@somaritane https://github.com/rancher/rancher/blob/362fb20d9e8ef0e2c779e1ee84300e8fda9eb316/chart/templates/deployment.yaml#L91 they will always have the same setting due to the configuration in our charts. Can you revert the new added changes? It should be fine